bonding.py

```diff
@@ -396,7 +398,7 @@ def peers(quiet=True):
     return groups
 
 
-def automated():
+def automated(peerswait=5):
```

Comment:
I originally had ``peerswait`` everywhere but the idea was ``peers``' ``wait`` parameter. 
But fair enough. I shouldn't get too clever with it.